webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
```

Comment:
I don't think we should recursively explore the stats graph, that could end up pulling in a lot of stats. For performance and implementation simplicity I think the selector should filter aggressively, only pulling in specific things. It would be hard to find all relationships recursively without producing all the stats, and we want to apply the filter before collecting, not after (or else the caller might as well filter from a full getStats). If we want other things too we could think about other types of selectors and unions of selectors. If a large graph is desired, a full getStats should be made.